Gemfile.lock

```diff
@@ -4,22 +4,22 @@ PATH
     high_voltage (2.4.0)
 
 GEM
-  remote: http://rubygems.org/
+  remote: https://rubygems.org/
   specs:
-    actionpack (4.2.0)
-      actionview (= 4.2.0)
-      activesupport (= 4.2.0)
-      rack (~> 1.6.0)
+    actionpack (4.2.5.2)
```

Comment:
Interesting, what's the reasoning behind ignoring the lock file? I haven't seen it ignored in any of our other open source libraries like Clearance or shoulda-matchers.